scripts/render-metrics-gif.py

```
@@ -191,6 +191,42 @@ def compute_global_limits(records: list[dict]) -> dict:
+    vals = [v for v in series if isinstance(v, (int, float))]
+    if not vals:
+        return " " * width
+    lo, hi = min(vals), max(vals)
+    span = hi - lo if hi > lo else 1.0
+    out = []
+    for v in series:
+        if not isinstance(v, (int, float)):
+            out.append(" ")
+            continue
+        # Quantise into 0..len(_SPARK_GLYPHS)-1 levels.
+        idx = int(round((v - lo) / span * (len(_SPARK_GLYPHS) - 1)))
```

Ruff (0.15.14)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)